src-js/instantsearch.js

```diff
@@ -24,7 +24,7 @@ export function init () {
         input: 'instantsearch-search'
       },
       placeholder: 'Search Documentation',
-      queryHook: debounce(function (query, searchFunction) { searchFunction(query); }, 500, { 'leading': true, 'trailing': true }) // method to throttle search requests
+      queryHook: debounce(function (query, searchFunction) { searchFunction(query); setTimeout(renderResults, 100); }, 500, { 'leading': true, 'trailing': true, 'maxWait': 1000 }) // method to throttle search requests
```

Comment:
There's probably a concern here that I'm not thinking of 🙈But why do we use `setTimeout` here?